application/client/src/app/forum/forum.component.html

```diff
@@ -27,6 +27,33 @@
         </div>
     </div>
     <!-- <button (click)="printItem(1)">Test It!</button> -->
+    <div *ngFor="let summaryThread_Prev of summaryThreadList" class="summaryThread shadow">
+        <div class="stIncidentIcon">
+            <img src={{summaryThread_Prev.iconpath}}  alt="Icon Image" class="stImageFormats">
+        </div>
+        <div class="stPestImage">
+            <img src={{summaryThread_Prev.imagepath}}  alt="Icon Image" class="stPestImageFormats">
+        </div>
+        <div class="stTitle">
+            <a routerLink="/expanded-discussion-view" class="stTitleLink">{{summaryThread_Prev.subject}}</a>
+        </div>
+        <div class="stDescription">
+            <span>{{summaryThread_Prev.comment.substring(0,90)}} ... </span>
+        </div>
+        <div class="stActivity">
+            <span>Views: {{summaryThread_Prev.record_count}} / Responses: {{summaryThread_Prev.record_count}}</span>
+        </div>
+        <div class="stRating">
+            <span>Positive Feedback: {{summaryThread_Prev.total_positive}} <span class="tab4"></span>Quality Rating: 
+                {{(summaryThread_Prev.total_positive / (summaryThread_Prev.record_count)) * 100 | number:'1.0-0'}}%</span>
```

Comment:
I also think the GET `summaryThreadList` query should be edited to pull the username instead of the creator ID, to match the sample thread summaries.

Reply:
I agree with both of these points! But either way, not a deal breaker for merging. So, yeah, @tomco21 when you get a chance make those updates but otherwise I am going to approve and merge